flagging_site/app.py

```diff
@@ -47,6 +47,28 @@ def create_app(config: Optional[Config] = None) -> Flask:
     from .data import db
     db.init_app(app)
 
+    @app.cli.command('get-postgres-usr')
+    @click.option('--user', prompt='Please enter your postgress user (usually this is postgres)', hide_input=False,
+              confirmation_prompt=False)
+    def some_function1(user):
+        click.echo('the postgres user is ... ' + user)
+
+    @app.cli.command('get-postgres-pwd')
+    @click.option('--password', prompt='Please enter your postgress pwd', hide_input=True,
+              confirmation_prompt=False)
+    def some_function(password):
+        click.echo('the password is ... ' + password)
```

Comment:
Can remove these as they don't do anything. (They don't save the user/password anywhere nor do they retrieve it from anywhere other than user inputs.)